pkg/cmd/attestation/verification/mock_verifier.go

```diff
@@ -27,6 +28,13 @@ func (v *MockSigstoreVerifier) Verify(attestations []*api.Attestation, policy ve
 		},
 		VerificationResult: &verify.VerificationResult{
 			Statement: statement,
+			Signature: &verify.SignatureVerificationResult{
+				Certificate: &certificate.Summary{
+					Extensions: certificate.Extensions{
+						BuildSignerURI: "https://github.com/foo/bar/.github/workflows/release.yml@refs/heads/main",
```

Comment:
You never know how URLs will get used: try to always use a namespace we control or something explicitly reserved for tests, i.e.

```suggestion
						BuildSignerURI: "https://github.com/github/example/.github/workflows/release.yml@refs/heads/main",
```

Reply:
(extremely minor nit, borne from fear of accidentally spamming real email addresses that were autogenerated 😉)